app/views/bikes/_bike.html.haml

```diff
@@ -1,32 +1,33 @@
 %li{ class: bike.stolen && "blist-stolen" }
   %a.blist-image-holder{href: bike_url(bike) }
```

Comment:
Also, it looks like the only place that is being rendered is fast_attr_update - to reduce confusion, delete this partial.

You can use the existing admin bike show on fast_attr_update, it'll be good enough

Reply:
https://github.com/bikeindex/bike_index/issues/636

Reply:
?